src/libmongoc/doc/debugging.rst

```diff
@@ -104,3 +104,9 @@ If you see "bson command installed by lldb_bson" at the beginning of your LLDB s
 
 .. _PyMongo: https://pypi.python.org/pypi/pymongo
 .. _install pip: https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py)
+
+Test assertions
+---------------
+
+To enable runtime test assertions, configure with ``-DENABLE_TEST_ASSERTIONS=ON``.
+``MONGOC_TEST_ASSERT`` is a noop if ``-DENABLE_TEST_ASSERTIONS=OFF``.
```

Comment:
Remove "MONGOC_TEST_ASSERT is a noop if -DENABLE_TEST_ASSERTIONS=OFF". `MONGOC_TEST_ASSERT` is a private macro, so users need not be made aware of it.